doc/source/jail-package.rst

```diff
@@ -1,42 +1,57 @@
-3. configure networking to enable internet access for this jail
-4. issue ``iocage record start UUID | TAG``, from now on every change will be recorded
-   under ``/iocage/jails/UUID/recorded``
-5. install nginx with ``pkg install nginx``
-6. install any other software you might require
-7. customize configuration files
-8. once finished, stop recording changes with ``iocage record stop UUID | TAG``
-   optionally stop jail
-9. examine ``/iocage/jails/UUID/recorded``, run ``find
-   /iocage/jails/UUID/recorded -type f``
-10. remove any unnecessary files, make final customization/changes
-11. run ``iocage package UUID | TAG``, this will create a package in
-    ``/iocage/packages`` with a SHA256 checksum file
-12. optionally discard the jail now with ``iocage destroy UUID | TAG``
-
-The resulting ``UUID.tar.xz`` can now be deployed on top of any new vanilla
+
+A jail package is a small differential image template. It can be
+deployed on top of vanilla jails. The RELEASE and patch level has to
+match between the package and a vanilla jail.
+
+:command:`iocage` uses the **record** function for this operation, which
+is a **unionfs** mount under the hood.
+
+The resulting package can be stored on a web server with a checksum
+file, ready to be deployed anywhere.
+
+Here are the steps to create a jail package:
+
+1. Create a new jail: :samp:`# iocage create -r 11.0-RELEASE tag=nginx`.
+2. Start the jail: :samp:`# iocage start nginx`.
+3. Configure networking to enable internet access for this jail.
+4. Issue :samp:`# iocage record start nginx`. From this point, every
+   change is recorded and added to :file:`/iocage/jails/[UUID]/recorded`.
+5. In the jail, install *nginx* with
+   :samp:`# iocage chroot nginx pkg install nginx`.
+6. In the jail, install any other required software.
+7. In the jail, customize the configuration files as needed.
+8. Once finished with installations and customizations, stop recording
+   changes with :command:`iocage record stop [UUID | TAG]`. Another
+   option is to stop the jail with :command:`iocage stop [UUID | TAG]`.
+9. Examine :file:`/iocage/jails/[UUID]/recorded` and run
+   :command:`find /iocage/jails/[UUID]/recorded -type f`.
+10. Remove unnecessary files and make any final customizations to the
+    jail.
+11. Run :command:`iocage package [UUID | TAG]`. This creates a new
+    package in :file:`/iocage/packages` with a *SHA256* checksum file.
+12. (Optional) Discard the jail with
+    :command:`iocage destroy [UUID | TAG]`.
+
+The resulting :file:`[UUID].tar.xz` is deployable on top of any new
 jail!
 
-1. create new jail ``iocage create -c``
-2. deploy package ``iocage import UUID tag=myjail``
-3. list jail ``iocage list|grep myjail``, grab UUID
-4. start jail ``iocage start UUID | TAG``
-5. examine your changes and packages - they are all there!
+Instructions to deploy a new jail with :file:`[UUID].tar.xz`.
 
-Enjoy!
+1. Create new jail: :samp:`# iocage create -r 11.0-RELEASE tag=myjail`.
```

Comment:
Gotcha - once the other reworks are done, I'll edit the index to remove this file and then nuke it completely.